HW05.Rmd

```diff
@@ -0,0 +1,153 @@
+
+  * compound_info shares the master_cpd_id variable with AUC_1 and AUC_2
+
+  * Experiment_info shares the experiment_id variable with AUC_1 and AUC_2
+
+  * cancer_cell_line_info shares the master_ccl_id variable with Experiment_info
+
+  
+```{r combine CTRP dataframes into one}  
+#Join AUC_1 and AUC_2
+AUC <- AUC_1 %>% #assign as own dataframe for posterity/easy-undo
+  full_join(AUC_2, by = c("experiment_id", "area_under_curve", "master_cpd_id"))
+
+#Join AUC and compound_info
+AUC_compound <- AUC %>% #assign as own dataframe for posterity/easy-undo
+  full_join(compound_info, by = "master_cpd_id")
+
+#Join AUC_compound and Experiment_info
+AUC_compound_Experiment <- AUC_compound %>% #assign as own dataframe for posterity/easy-undo
+  full_join(Experiment_info, by = c("experiment_id" = "expt_id"))  
+
+#final join to put all together
+full_combo <- AUC_compound_Experiment %>% #assign as own dataframe for posterity/easy-undo
+  full_join(cancer_cell_line_info, by = "master_ccl_id")
+```
+
+## ANSWER THE FOLLOWING QUESTIONS (WITH GRAPHS!)
+
+**Which cancer type has the lowest AUC values to the compound "vorinostat"?**
+
+```{r solve vorinostat}
+#cpd_name will have to be "vorinostat"
+vorinostat_search <- full_combo %>% #assign to dataframe vorinostat_search
+  filter(cpd_name == "vorinostat") %>% #only look at vorinostat compound rows
+  arrange(area_under_curve) #order so that smallest AUC at top
+
+vorinostat_search %>%
+  ggplot() + #plots plots plots plots plots plots EVERYBODY
+  aes(area_under_curve, cancer_type) + #look at AUC by cancer_type
+  geom_boxplot(color = "dodgerblue1", fill = "dodgerblue3") + #boxplots to demo mean vs outlier distinction
+  labs(x = "AUC", #area_under_curve
+       y = "Cancer Type", #cancer_type
+       title = "Vorinostat AUC by Cancer Type",
+       subtitle = "On average, it appears haematopoietic\nand lymphoid tissue have the lowest AUC") +
+  theme(plot.title = element_text(hjust = 0.5)) + #center title
+  theme(plot.subtitle = element_text(hjust = 0.5)) #center subtitle
+```
+
+Based on this plot, it seems on average haematopoietic and lymphoid tissue have the lowest AUC
+However, upper aerodigestive tract has that one super low AUC outlier, of note
+
+
+
+
+**Which compound is the prostate cancer cell line 22RV1 most sensitive to? (For 22RV1, which compound has the lowest AUC value?)**
+
+```{r solve 22RV1}
+line22RV1_search <- full_combo %>% #assign to dataframe line22RV1_search
+  filter(ccl_name == "22RV1") %>% #only look at ccl_name of 22RV1
+  arrange(area_under_curve) #order so that smallest AUC at top
+
+line22RV1_search %>% #use this dataframe to plot
+  slice_head(n = 10) %>% #I don't care about the ones with large AUCs. Top 10 smallest pls
+  ggplot() + #plots plots plots plots plots plots EVERYBODY
+  aes(area_under_curve, cpd_name) + #look at AUC by cpd_name
+  geom_point(color = "orange") + #point graph
```

Comment:
The scatterplot here is not very intuitive. A bar plot may be a better option in this case.